kong/runloop/certificate.lua

```diff
@@ -119,15 +119,23 @@ local function fetch_sni(sni, i)
 end
 
 
-local function fetch_certificate(sni)
-  local certificate, err = singletons.db.certificates:select(sni.certificate)
+local function fetch_certificate(pkey, sni_name)
```

Comment:
I was confused by this variable name: `pkey` is frequently used to mean "private key" in certificate handling code.

Reply:
Agreed, we use `pk` or something similar in our DAO code when we mean "primary key". We should not use `pkey` here (I too got confused)

Reply:
Fixed. Thanks for pointing out!